services/common/src/constants/assets.ts

```diff
@@ -1 +1,3 @@
 export { default as SMALL_PIN_SELECTED } from "../assets/images/small-pin-selected.svg";
+// export { default as FLUSH_SOUND } from "../assets/downloads/flush.mp3";
```

Comment:
saving these for when we restore the flush in the component you moved to the common?